RoomDraw/app/models/draw_group.rb

```diff
@@ -57,7 +59,10 @@ def calculate_draw_num
   end
```

Comment:
Line 57: I don't think we should be subtracting 10k from the mins of non-suite groups.